org.thonny.Thonny.yaml

```diff
@@ -0,0 +1,92 @@
+---
+id: org.thonny.Thonny
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+command: thonny
+cleanup:
+  - /lib/debug
+  - /man
+  - /share/man
+finish-args:
+  # Thonny requires access to serial / USB devices for embedded development.
+  - --device=all
+
+  # Ensure that extra Python packages, such as Thonny plugins, are installed to a location inside the Flatpak.
+  - --persist=.local/lib
+
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+modules:
+  - name: tcl
+    buildsystem: autotools
+    subdir: unix
+    build-options:
+      no-debuginfo: true
+    cleanup:
+      - /bin
+      - /lib/debug
+      - /lib/pkgconfig
+      - /man
+    sources:
+      - type: archive
+        url: https://prdownloads.sourceforge.net/tcl/tcl8.6.11-src.tar.gz
+        sha256: 8c0486668586672c5693d7d95817cb05a18c5ecca2f40e2836b9578064088258
+  - name: tk
+    buildsystem: autotools
+    subdir: unix
+    build-options:
+      no-debuginfo: true
+    cleanup:
+      - /bin
+      - /lib/debug
+      - /lib/pkgconfig
+      - /man
+    sources:
+      - type: archive
+        url: https://prdownloads.sourceforge.net/tcl/tk8.6.11.1-src.tar.gz
+        sha256: 006cab171beeca6a968b6d617588538176f27be232a2b334a0e96173e89909be
+
+  # Derived from how the Freedesktop Sdk bundles Python: https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/master/elements/components/python3.bst
+  - name: python
+    buildsystem: autotools
+    config-opts:
+      - --enable-optimizations
+      - --with-ensurepip=install
+      - --enable-shared
+      - --with-system-expat
+      - --with-system-ffi
+      - --enable-loadable-sqlite-extensions
+      - --with-dbmliborder=gdbm
+    cleanup:
+      - /bin/idle*
+      - /lib/python3.8/idlelib
+      - /lib/python3.8/test
+      - /lib/python3.8/turtle*
+      - /lib/python3.8/__pycache__/turtle.*
+      - /lib/python3.8/*/test
+      - /lib/python3.8/*/tests
+    cleanup-commands:
+      - find "/lib" -name "lib*.a" -exec rm {} ";"
+    sources:
+      - type: archive
+        url: https://www.python.org/ftp/python/3.8.8/Python-3.8.8.tar.xz
```

Comment:
do you really need to build python?

Reply:
Is there a better way to include the `tkinter` module bundled with Python? The module is deleted in the org.freedesktop.Sdk.

Reply:
Seems like there is an open issue:
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/976

Reply:
and there is a solution there: https://github.com/flathub/flathub/pull/2146/commits/5f6f8d8b63fc623c8eda6b221fd6e40851e9259f

Reply:
or tl;dr https://github.com/iwalton3/tkinter-standalone

Reply:
Awesome! That helps a lot.